app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    file1 = command.split(" ")[1]
+    file2_path = command.split(" ")[2]
+
+    for i in range(file2_path.count("/")):
+        path = "/".join(file2_path.split("/")[:i + 1])
+        if os.path.exists(path):
+            continue
+        os.mkdir(path)
```

Comment:
Will be better to use os module functions `os.path.split()` to separate dir name and  `os.makedirs()` to create all dir tree